src/components/structures/GroupView.js

```diff
@@ -428,7 +428,7 @@ export default createReactClass({
         };
     },
 
-    componentWillMount: function() {
+    componentDidMount: function() {
         this._unmounted = false;
         this._matrixClient = MatrixClientPeg.get();
         this._matrixClient.on("Group.myMembership", this._onGroupMyMembership);
```

Comment:
two lines below this, `_changeAvatarComponent ` looks entirely unused (looked ref-like but can't see any uses)
note for your further stages of these PRs

Reply:
A bunch of this is broken out in the 'random code cleanup' PR already reviewed.